src/Reviews.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default function Reviews({ reviews }) {
+  if (!reviews || !reviews.length) {
+    return (
+      <p>
+        등록된 리뷰가 없습니다!
+      </p>
+    );
+  }
+
+  return (
+    <>
+      <h3>리뷰</h3>
+      <ul>
+        {reviews.map(({
+          id, name, score, description,
+        }) => (
+          <li key={id}>
+            <div>
+              작성자:
+              {' '}
+              {name}
+            </div>
+            <div>
+              평점:
+              {' '}
+              {score}
+            </div>
+            <div>
+              리뷰 내용:
+              {' '}
+              {description}
+            </div>
+          </li>
+        ))}
+      </ul>
+    </>
+  );
```

Comment:
부정문으로 예외처리하는건 동료들에게 한번 더 고민하게 만드는 것 같아요. 예외 처리의 순서를 조금 바꾸는건 어떨까요? 

그리고 조금 더 명확하게 처리해줘도 좋겠네요. 아래와 같은 방식은 어떤가요?

```suggestion
  return (
    <>
      <h3>리뷰</h3>
      {reviews && reviews.length > 0 ? (
      <ul>
        {reviews.map(({
          id, name, score, description,
        }) => (
          <li key={id}>
            <div>
              작성자:
              {' '}
              {name}
            </div>
            <div>
              평점:
              {' '}
              {score}
            </div>
            <div>
              리뷰 내용:
              {' '}
              {description}
            </div>
          </li>
        ))}
      </ul>
      ) : (
      <p>
        등록된 리뷰가 없습니다!
      </p>
      )
    </>
  );
```

Reply:
아하!!  생각해보니 회사에서도 작업하면서 부정문보다는 있는 그대로 예외처리를 해주려고 노력하는 데 코드숨 과제에도 적용시켰으면 더 좋았을 거 같네요 ㅎㅎ